### PR TITLE
Creating review with rating and feedback

### PR DESCRIPTION
**Pull request by sevelinCa**

### CODE REVIEW

**Review by iChris-tian — CHANGES_REQUESTED** [review]
Make changes in the database config

**Review by sevelinCa — COMMENTED** [review]
I was done to change it

**Review by Ismaelmurekezi — APPROVED** [review]
Nice work! for me it good to go. Approved

File: src/controllers/review.controller.ts
Comment (iChris-tian) [inline src/controllers/review.controller.ts:1]: change 'bougth' to 'bought'

**Review by iChris-tian — CHANGES_REQUESTED** [review]
Just a small minor change

**Review by iChris-tian — APPROVED** [review]
ok

**Pull request merged**